### DISCUSSION TITLE
Batteriefüllstand im Display

### DISCUSSION THREAD

**Discussion by kappo76** [hoylabs/OpenDTU-OnBattery#1369] [General]
Hallo,
könnt ihr bitte den Batteriefüllstand in % in die Displayanzeige integrieren.

Gruß
Kappo

**Comment by Tiese** [comment]
Hallo @kappo76,

hier ist es integriert: [http://opendtu-client.kurze.email](http://opendtu-client.kurze.email)

Schöne Grüße
Tiese
